lib/resources/aws/aws_s3_bucket_object.rb

```diff
@@ -0,0 +1,78 @@
+# author: Matthew Dromazos
+class AwsS3BucketObject < Inspec.resource(1)
+  name 'aws_s3_bucket_object'
+  desc 'Verifies settings for a s3 bucket object'
+  example "
+    describe aws_s3_bucket_object(bucket_name: 'bucket_name', key: 'file_name') do
+      it { should exist }
+      it { should_not be_public }
+    end
+  "
+  supports platform: 'aws'
+
+  include AwsSingularResourceMixin
+  attr_reader :bucket_name, :key
+
+  def to_s
+    # keep the format that aws uses.
+    "s3://#{@bucket_name}/#{@key}"
+  end
+
+  def object_acl
+    @object_acl ||= BackendFactory.create(inspec_runner).get_object_acl(bucket: bucket_name, key: key).grants
```

Comment:
You'll need to break this one-liner apart a bit, and wrap the get_object_acl call in a block:

```
return @object_acl if defined? @object_acl
catch_aws_errors do
  @object_acl = BackendFactory.create(inspec_runner).get_object_acl(bucket: bucket_name, key: key).grants
end
@object_acl
```